packages/tasks_repository/tasks_data.dart

```diff
@@ -1,30 +1,17 @@
-class Task {
-final String title;
-final String description;
-final bool isDone;
+import 'package:freezed_annotation/freezed_annotation.dart';
 
-Task({
-required this.title,
-required this.description,
-required this.isDone,
-});
-}
+part 'tasks_data.freezed.dart';
+part 'tasks_data.g.dart';
 
-List tasks = [
-Task(
-title: 'Task 1',
-description: 'Lear Flutter',
-isDone: false,
-),
-Task(
-title: 'Task 2',
-description: 'Clean room',
-isDone: true,
-),
-Task(
-title: 'Task 3',
-description: 'Go to the gym',
-isDone: false,
-),
-];
+@freezed
 
+/// Task class autogenerated by freezed
+class Task with _$Task {
+  const factory Task({
+    required String description,
+    required bool isDone,
+  }) = _Task;
+
+/// get Task from json
+  factory Task.fromJson(Map<String, Object?> json) => _$TaskFromJson(json);
+}
```

Comment:
Wait. Why do you have Task model in this file?? 
Task model should be in models/task.dart file? DO you have it there?